frontend/components/SetupPage/Create/SetupSeedPhrase.tsx

```diff
@@ -1,29 +1,75 @@
 import { CopyOutlined } from '@ant-design/icons';
-import { Button, Card, Flex, message, Tag, Typography } from 'antd';
+import { Button, Card, Flex, message, Modal, Tag, Typography } from 'antd';
+import { useCallback, useState } from 'react';
 
+import { CustomAlert } from '@/components/Alert';
 import { SetupScreen } from '@/enums/SetupScreen';
 import { useSetup } from '@/hooks/useSetup';
 import { copyToClipboard } from '@/utils/copyToClipboard';
 
 import { SetupCreateHeader } from './SetupCreateHeader';
 
+const { Text, Title } = Typography;
+
+const SeedPhraseAlert = () => (
+  <CustomAlert
+    type="warning"
+    showIcon
+    message={
+      <Text>
+        Without seed phrase, access to your Pearl account will be lost forever,
+        so keep it safe.
+      </Text>
+    }
+  />
+);
+
 export const SetupSeedPhrase = () => {
   const { mnemonic, goto } = useSetup();
+  const [hasCopied, setHasCopied] = useState(false);
+  const [modal, contextHolder] = Modal.useModal();
 
-  const handleNext = () => {
-    goto(SetupScreen.SetupBackupSigner);
-  };
+  const onCopy = useCallback(() => {
```

Comment:
on* should be reserved for props exposed by the component, i.e. <button onClick={}/>
```suggestion
  const handleCopy = useCallback(() => {
```

handlers are the functions that handle the events after some on* event occurs